.github/workflows/bump-app-version.yml

```diff
@@ -0,0 +1,37 @@
+name: Bump App Version
+
+on:
+  workflow_dispatch:
+  push:
+    # trigger on new minor release tags only
+    tags:
+      - valora-v1.[0-9]+.0
+
+jobs:
+  bump-app-version:
+    # this job uses the pre-deploy script and agvtool which is only available on macos
+    runs-on: macos-11
+    permissions:
+      pull-requests: write
+    env:
+      VALORA_BOT_PAT: ${{ secrets.VALORA_BOT_PAT }}
```

Comment:
i kept this personal access token in the GH secrets instead of google cloud because it expires every few months so seemed easier to update from here, but i can move it to gcp if anyone objects?

Reply:
It seems we wanted to have all secrets in GCP.
I'll let @silasbw confirm.